analysers/trials/cpu.py

```diff
@@ -52,7 +52,7 @@ def computeMetrics(data):
     dataIntegral = integrate.trapz(data).item()
```

Comment:
The changes are already in: https://github.com/benchflow/analysers/pull/51/files/308f5db903e9ea4515c647018d29c87835aa5db6..f8a0f6ddf9ed80eec8ce9b5d8e95f1a68fd2b363